crates/rspack_core/src/dependency/const_dependency.rs

```diff
@@ -34,11 +37,31 @@ impl CodeGeneratable for ConstDependency {
     }
 
     let expr = self.expression.clone();
-    cgr.visitors.push(
-      create_javascript_visitor!(exact &self.ast_path, visit_mut_expr(n: &mut Expr) {
-        *n = expr.clone();
-      }),
-    );
+
+    let mut is_member = false;
+    if let Some(path) = self.ast_path.last() {
```

Comment:
This code seems hard to read, here are some suggested solutions
* check member expr in `scanner -> visit_expr` and const_dependency no need to change
* or add a type field in const_dependency

cc @underfin

Reply:
> check member expr in scanner -> visit_expr and const_dependency no need to change

I think this is better.